[17.0][ADD] disable_odoo_online_menu

vincent-hatakeyama opened the pull request:
Split disable_odoo_online in two modules so it does not forces to install mail to gain their features.

Splitting the feature in two is necessary for a customer that does not install the mail module and does not want to install it but want the other features of the module.

pedrobaeza reviewed (changes requested):
The module is so small and the case is so specific that I don't think this should be done in OCA, but on a custom module, and more being in a stable version that switching dependencies will break some things.

vincent-hatakeyama committed:
[ADD] disable_odoo_online_menu

Split disable_odoo_online in two modules so it does not forces to
install mail to gain their features.

legalsylvain commented:
hi @vincent-hatakeyama. I think that a less intrusive change is just : 

 
- keep the disable_odoo_online module 
- create a new module named disable_odoo_online_mail depends on disable_odoo_online and mail, auto installable. 
- move the "mail" feature in the dedicated module, and remove dependency in disable_odoo_online.

What do you think ? 

Such refactor could be done in V19 that will be out in a few days;
